docs/HowToAddnewBoard.md

```diff
@@ -170,32 +170,33 @@ endif
 ## How to include arch and board header files
 
 At build time, Makefile dynamically links selected chip and board folders into selected cpu folder.  
-After building, you can find *chip* and *board* folders at *os/arch/<cpu-familyname>/src* folder.
+After building, you can find *chip* and *board* folders at *os/arch/\<cpu-familyname\>/src* folder.
 
 Additionally, at that time, *include* folder is linked into *os/include/arch* folder to expose their public APIs and definitions.
 
 For example, to use *bcm4390x* chip API which is defined in *os/arch/arm/include/bcm4390x/chip.h*, `#include <arch/chip/chip.h>` is necessary.  
 For *os/board/artik05x/artik055_alc5658_i2c.h*, `#include <arch/board/artik055_alc5658_i2c.h>` can be used.
 
-*make distclean* command unlinks them.
+>**Note**
+>If you want to unlink the chip and board folders from the linked cpu folder, you can use the *make distclean* command.
 
 ## Configs
 
 Changes in *configs* folder includes build options and flash operations.
 
 1. Add a new folder with board name.
 ```
-mkdir build/configs/<board-name>
+	mkdir build/configs/<board-name>
```

Comment:
Do you want to add spaces for bash commands? Then, let's apply at frist README.

Reply:
For numbered lists, I think the review comment suggests indenting BASH commands. However, in my opinion, if the BASH lines are too long, sometimes, not indenting is preferred for better readability. You are right, we need to do similarly for README, thanks for spotting that.